extensions-core/avro-extensions/pom.xml

```diff
@@ -68,6 +76,11 @@
       <version>${schemarepo.version}</version>
     </dependency>
     <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-schema-registry-client</artifactId>
```

Comment:
hola @himanshug, as seen with command `mvn -pl extensions-core/avro-extensions dependency:tree`, the `kafka-schema-registry-client` only pulls `org.slf4j:slf4j-log4j12:jar:1.7.6` transitive dependency.